packages/carbon-graphs/src/js/controls/Bubble/BubbleMultipleDataset.js

```diff
@@ -107,11 +107,10 @@ class BubbleMultipleDataset extends Bubble {
     if (position > -1) {
       graph.config.shownTargets.splice(position, 1);
     }
+    const bubble =  graph.content.filter((bubble) => bubble.config.key === graphData.key);
```

Comment:
Other graph controls have a check to validate that the Array is populated before calling reflowLegend.  Is that not necessary here and in the BubbleSingleDataset?

Reply:
I had missed it. Updated here https://github.com/cerner/terra-graphs/pull/61/commits/2c48856482a6cf65cc690acbfd2ff06bb77fce25.